plugins/inputs/cisco_telemetry_mdt/sample.conf

```
@@ -21,6 +21,8 @@
  ## Define (for certain nested telemetry measurements with embedded tags) which fields are tags
  # embedded_tags = ["Cisco-IOS-XR-qos-ma-oper:qos/interface-table/interface/input/service-policy-names/service-policy-instance/statistics/class-stats/class-name"]
 
+  ## Include the delete field in every telemetry message.
+  # include_delete_field = false
```

Maybe add a newline to separate this option from the rest of the config
```suggestion
  ## Include the delete field in every telemetry message.
  # include_delete_field = false
  
```